pages/about/code-of-conduct.vue

```diff
@@ -38,20 +38,97 @@
+                                    sm:h-40
+                                    bg-contain bg-no-repeat
+                                    mx-2
+                                    md:mx-5
+                                    my-10
+                                    sm:my-5
+                                    flex
+                                    justify-center
+                                "
+                                :style="getSub1Style(img)"
+                            >
+                                <span class="img_text">
+                                    {{ rule.list[j] }}
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                    <div
+                        v-for="(subtitle, subtitleIndex) in rule.subtitle"
+                        :key="`coc-description-${subtitleIndex}`"
+                    >
+                        <div>
+                            <two-col-wrapper>
+                                <template #default>
+                                    <h2>{{ subtitle.title }}</h2>
+                                </template>
+                                <template #right-col>
+                                    <div
+                                        v-for="(
+                                            description, index
+                                        ) in subtitle.descriptions"
+                                        :key="'description-' + index"
+                                    >
+                                        <p>{{ description }}</p>
+                                        <br
+                                            v-if="
+                                                index !==
+                                                subtitle.descriptions.length - 1
+                                            "
+                                        />
+                                    </div>
+                                    <ol class="list-decimal ml-4">
+                                        <li
+                                            v-for="(
+                                                li, liIndex
+                                            ) in subtitle.list"
+                                            :key="`coc-list-${liIndex}`"
+                                        >
+                                            {{ li }}
+                                        </li>
+                                    </ol>
+                                </template>
+                            </two-col-wrapper>
+                            <!-- 處理流程圖片 -->
+                            <div
+                                v-if="
+                                    [`處理流程`, `Procedure`].includes(
+                                        subtitle.title,
+                                    )
+                                "
```

Comment:
**suggestion:** Ditto, please help us add the comments here to explain the meaning of the condition

Reply:
OK

Reply:
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/40282726/224529467-39da3da2-29eb-45c3-8ba8-c215ede186bc.png">